### PR TITLE
adding criteria convert video

### PR DESCRIPTION
**Pull request by LimbertVargas**
hello guys, plkease check the criteria convert video class and reviewed please

### CODE REVIEW

File: src/main/java/com/fundation/webservice/model/VideoConvert.java [hunk 1]
```diff
@@ -25,65 +25,34 @@
  * @version 1.0
  */
 public class VideoConvert implements IConvert {
-    private String inputFile;
-    private String outputFile;
-    private String newFormat;
-
```
Comment (MadayAlcala) [inline src/main/java/com/fundation/webservice/model/VideoConvert.java]: Is this space necessary?

File: src/main/java/com/fundation/webservice/model/VideoConvert.java [hunk 2]
```diff
@@ -25,65 +25,34 @@
...
-    //audio
-    private String aCodec;
-    private int aBit;
-    private int aChannel;
-    private int aRate;
-
```
Comment (MadayAlcala) [inline src/main/java/com/fundation/webservice/model/VideoConvert.java]: Is this space neccessary?

File: src/main/java/com/fundation/webservice/model/VideoConvert.java [hunk 3]
```diff
@@ -25,65 +25,34 @@
...
-        this.aCodec = aCodec;
-        this.aBit = aBit;
-        this.aChannel = aChannel;
-        this.aRate = aRate;
-
-        //video
-        this.vCodec = vCodec;
-        this.vBit = vBit;
-        this.vTag = vTag;
-        this.vRate = vRate;
```
Comment (MadayAlcala) [inline src/main/java/com/fundation/webservice/model/VideoConvert.java]: Is this space necessary?

File: src/main/java/com/fundation/webservice/model/VideoConvert.java [hunk 4]
```diff
@@ -25,65 +25,34 @@
...
+    public VideoConvert(CriteriaVideo criterion){
+        this.criterion=criterion;
     }
 
     public void convert() {
         try {
-            File source = new File(inputFile);
-            File target = new File(outputFile);
+            File source = new File(criterion.getSrcPath());
+            File target = new File(criterion.getDestPath());
```
Comment (MadayAlcala) [inline src/main/java/com/fundation/webservice/model/VideoConvert.java]: Is this space necessary?

File: src/main/java/com/fundation/webservice/model/VideoConvert.java [hunk 5]
```diff
@@ -25,65 +25,34 @@
...
             //Audio Attributes
             AudioAttributes audio = new AudioAttributes();
-            audio.setCodec(aCodec);
-            audio.setBitRate(aBit);
-            audio.setChannels(aChannel);
-            audio.setSamplingRate(aRate);
+            audio.setCodec(criterion.getaCodec());
+            audio.setBitRate(criterion.getaBit());
+            audio.setChannels(criterion.getaChannel());
+            audio.setSamplingRate(criterion.getaRate());
```
Comment (MadayAlcala) [inline src/main/java/com/fundation/webservice/model/VideoConvert.java]: Is this space necessary?

File: src/main/java/com/fundation/webservice/model/VideoConvert.java [hunk 6]
```diff
@@ -25,65 +25,34 @@
...
             //Video Attributes
             VideoAttributes video = new VideoAttributes();
-            video.setCodec(vCodec);
-            video.setTag(vTag);
-            video.setBitRate(new Integer(vBit));
-            video.setFrameRate(new Integer(vRate));
+            video.setCodec(criterion.getvCodec());
+            video.setTag(criterion.getvTag());
+            video.setBitRate(new Integer(criterion.getvBit()));
+            video.setFrameRate(new Integer(criterion.getvRate()));
```
Comment (MadayAlcala) [inline src/main/java/com/fundation/webservice/model/VideoConvert.java]: Is this space necessary?

File: src/main/java/com/fundation/webservice/model/VideoConvert.java [hunk 7]
```diff
@@ -25,65 +25,34 @@
...
 
             //Encoding attributes
             EncodingAttributes attrs = new EncodingAttributes();
-            attrs.setFormat(newFormat);
+            attrs.setFormat(criterion.getNewFormat());
             attrs.setAudioAttributes(audio);
             attrs.setVideoAttributes(video);
```
Comment (MadayAlcala) [inline src/main/java/com/fundation/webservice/model/VideoConvert.java]: Is this space necessary?

File: src/main/java/com/fundation/webservice/model/VideoConvert.java [hunk 8]
```diff
@@ -25,65 +25,34 @@
...
-
-        //video
-        this.vCodec = vCodec;
-        this.vBit = vBit;
-        this.vTag = vTag;
-        this.vRate = vRate;
 
+    private CriteriaVideo criterion;
+    public VideoConvert(CriteriaVideo criterion){
+        this.criterion=criterion;
```
Comment (MadayAlcala) [inline src/main/java/com/fundation/webservice/model/VideoConvert.java]: Please apply spaces before and after the '='.

File: src/main/java/com/fundation/webservice/model/VideoConvert.java (hunk 3 above)
Comment (LimbertVargas) [reply src/main/java/com/fundation/webservice/model/VideoConvert.java]: resolve

File: src/main/java/com/fundation/webservice/model/VideoConvert.java (hunk 4 above)
Comment (LimbertVargas) [reply src/main/java/com/fundation/webservice/model/VideoConvert.java]: resolve

File: src/main/java/com/fundation/webservice/model/VideoConvert.java (hunk 5 above)
Comment (LimbertVargas) [reply src/main/java/com/fundation/webservice/model/VideoConvert.java]: resolve

File: src/main/java/com/fundation/webservice/model/VideoConvert.java (hunk 6 above)
Comment (LimbertVargas) [reply src/main/java/com/fundation/webservice/model/VideoConvert.java]: resolve

File: src/main/java/com/fundation/webservice/model/VideoConvert.java (hunk 7 above)
Comment (LimbertVargas) [reply src/main/java/com/fundation/webservice/model/VideoConvert.java]: resolve

**Pull request merged**